test/unit/specs/store/json/txs.js

```diff
@@ -10,28 +10,9 @@ export default [
           {
             type: `cosmos-sdk/Send`,
             value: {
-              inputs: [
-                {
-                  coins: [
-                    {
-                      denom: `jbcoins`,
-                      amount: `1234`
-                    }
-                  ],
-                  address: `tb1da6xsetjg9jxgun9wdesexv05j`
-                }
-              ],
-              outputs: [
-                {
-                  coins: [
-                    {
-                      denom: `jbcoins`,
-                      amount: `1234`
-                    }
-                  ],
-                  address: `tb1d4u5zerywfjhxuc9nudvw`
-                }
-              ]
+              from_address: `tb1da6xsetjg9jxgun9wdesexv05j`,
```

Comment:
I don't want to be too picky but the prefix should be `cosmos` instead of `tb` in all mocked addresses

Reply:
Here I tried to keep the snapshot identical by changing only the way the message was built, but we can adhere to the `cosmos` prefix, or have just X, Y, Z as addresses so it's easier to follow the tests

Reply:
I try to propose the A, B addresses, let me know what do you think